Library/ViewModels/ConfirmDetailsViewModel.swift

```diff
@@ -313,10 +317,51 @@ public class ConfirmDetailsViewModel: ConfirmDetailsViewModelType, ConfirmDetail
     }
 
     self.configureCTAWithPledgeTotal = Signal.combineLatest(project, pledgeTotal)
+
+    // MARK: CreateCheckout GraphQL Call
+
+    let pledgeDetailsData = Signal.combineLatest(
+      project,
+      rewards,
+      pledgeTotal,
+      refTag
+    )
+
+    let createCheckoutEvents = pledgeDetailsData
+      .takeWhen(self.continueCTATappedProperty.signal)
+      .map { project, rewards, pledgeTotal, refTag in
+        let rewardsIDs = rewards.map { $0.graphID }
+
+        return CreateCheckoutInput(
+          projectId: project.graphID,
+          amount: "\(pledgeTotal)",
```

Comment:
Presumably `pledgeTotal` is a double - does it need to be formatted in a specific way for the server? My concern would be that the automatic printer might output "7" or "7.0" or "7.000000001" and the server might expect a specific number of decimal places.